tests/models/test_game.py

```diff
@@ -1,13 +1,20 @@
-from datetime import datetime
-
 import pytest
-from mars_api.models import Game
 
+from mars_api.models import MAPS, Game
+
+pytestmark = pytest.mark.django_db
+
+
+def test_game_cannot_be_created_with_invalid_game_map():
+    pytest.fail("unimplemented")
```

Comment:
Look into `skip` and `xfail` as well

Reply:
My current "convention" is to use `fail` for particularly important tests, and `skip` for not as important ones